src/main/scala/com/exasol/cloudetl/helper/ParquetSchemaConverter.scala

```diff
@@ -96,7 +97,14 @@ final case class ParquetSchemaConverter(isLowercaseSchemaEnabled: Boolean) exten
       case `jBoolean`      => getPrimitiveType(columnName, BOOLEAN, repetition, None, None)
       case `jSqlDate`      => getPrimitiveType(columnName, INT32, repetition, None, Option(dateType()))
       case `jSqlTimestamp` => getPrimitiveType(columnName, INT96, repetition, None, None)
-      case _ => throw new IllegalArgumentException(s"Cannot convert Exasol type '$columnType' to Parquet type.")
+      case _ =>
+        throw new IllegalArgumentException(
+          ExaError
+            .messageBuilder("E-CSE-22")
+            .message("Cannot convert Exasol type {{TYPE}} to Parquet type.")
```

Comment:
is that something we expect to happen?
If yes:
Add a mitiogation.
If no:
Maybe you can change type to F- and add `.ticketMitigation()`